docs/cli-usage.md

```diff
@@ -87,7 +87,7 @@ djinni \
 | `--jni-include-prefix <prefix>` | The prefix for `#includes` of JNI header files from JNI C++ files. |
 | `--jni-include-cpp-prefix <prefix>` | The prefix for `#includes` of the main header files from JNI C++ files. |
 | `--jni-namespace ...` | The namespace name to use for generated JNI C++ classes. |
-| `--jni-base-lib-include-prefix ...` | The JNI base support library's include path (default: `djinni/jni/`). |
+| `--jni-base-lib-include-prefix ...` | The JNI base support library's include path. |
```

Comment:
I think we just had that  due to backward compatibility, and since we changed the include to the whole path (e.g. djinni/jni/Marshal.hpp) it's defacto not required.
I just wonder, shall we not remove the option, or is there still a use case for it?

Reply:
That's a good question. In our project we have the support lib as a submodule, so the include path is flexible. I don't think it's much different if you're installing the support lib instead. To be honest, I tend to agree it's a good opportunity to clean up some of these options. Should we create a separate PR and get some feedback from the community on Slack before removing those?